backend/classes/Game.js

```diff
@@ -0,0 +1,21 @@
+
+class Game {
+  constructor(id) {
+    this.id = id;
+
+    this.playerList = {};
+
+    //Phase mean what is currently happen in the game - waiting for players, murderer Input? etc
```

Comment:
that is obvious - try to avoid comments as much as possible. if you need the comment, then the code is not clear

Reply:
deleted